docs/UserGuide.adoc

```diff
@@ -62,27 +62,32 @@ Format: `help`
 === Adding a person: `add`
 
 Adds a person to the list of people +
-Format: `add n/NAME ic/NRIC p/PHONE_NUMBER e/EMAIL a/ADDRESS b/BIRTHDAY [pol/POLICY] [t/TAG]…​`
+Format: `add n/NAME ic/NRIC p/PHONE_NUMBER e/EMAIL a/ADDRESS b/BIRTHDAY [pol/POLICY]... [t/TAG]...​`
 
 ****
-* A person can have any number of tags (including 0)
+* A person can have any number of tags (including 0).
+* A person can have any number of policies (including 0).
+* Birthdays are in the form `DD.MM.YYYY`.
 ****
 
 Examples:
 
-* `add n/John Doe ic/S9999999J p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/12.09.1980 pol/SeniorCare t/high-priority`
+* `add n/John Doe ic/S9999999J p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/12.09.1980 pol/1 pol/2 t/diabetic t/smoker`
 
 === Adding a policy: `addpolicy`
 
 Adds a policy to the list of policies +
-Format: `addpolicy  add pol n/NAME d/DESCRIPTION c/TIME_PERIOD p/PRICE [sa/START_AGE] [ea/END_AGE]... [t/TAGS]...`
+Format: `addpolicy n/NAME d/DESCRIPTION c/TIME_PERIOD p/PRICE sa/START_AGE ea/END_AGE [t/TAGS]... [cri/CRITERIA]...`
 
 ****
-* A policy can have any number of tags (including 0)
+* A policy can have any number of tags (including 0).
+* A policy can have any number of criteria (including 0).
+* Coverage time period is specified in years.
+* Price is specified in dollar ($) units.
 ****
 Examples:
 
-* `add pol n/SeniorCare d/care for seniors c/10months p/$50000 [sa/50 ea/75]`
+* `addpolicy n/SeniorCare d/Care for seniors c/5 p/50000 sa/50 ea/75]`
```

Comment:
oh could you change the c/ part? cos I was thinking we need to distinguish the number of days, months and years so maybe can do something like c/years/5? so like if there are months or days specified then it'll be c/days/20 months/11 years/5 or something. does that make sense?

Reply:
Okay yeah that makes sense. Made the adjustments in my commit below, thanks :)